_sub/compute/k8s-traefik/vars.tf

```diff
@@ -44,4 +44,47 @@ variable "request_memory" {
   type = string
   description = "(optional) Describes the minimum amount of memory required"
   default = "128Mi"
-}
\ No newline at end of file
+}
+
+variable "dashboard_deploy" {
+  type        = bool
+  description = "Deploy ingress for secure access to Traefik dashboard."
+  default     = false
+}
+
+variable "dashboard_username" {
+  type        = string
+  description = "Username used for basic authentication."
+  default     = "cloudengineer"
+}
+
+variable "dashboard_secret_name" {
+  type        = string
+  description = "Name of the k8s secret to store the credentials for basic authentication."
+  default     = "traefik-basic-auth"
+}
+
+variable "dashboard_ingress_name" {
+  type        = string
+  description = "Name of the ingress, must be unique."
+  default     = "traefik-dashboard"
```

Comment:
This risk clashing with additional Traefik instances. Should this contain the deploy_name, e.g. ${var.deploy_name}-dashboard?

Reply:
Fixed.